packages/testsweets/lib/src/ui/shared/widgets/inspect_layout_widget.dart

```diff
@@ -13,64 +13,62 @@ class InspectLayoutView extends ViewModelWidget<WidgetCaptureViewModel> {
 
   @override
   Widget build(BuildContext context, WidgetCaptureViewModel model) {
-    return Expanded(
-      child: Stack(
-        children: [
-          Positioned.fill(
-            child: Container(
-              width: double.infinity,
-              height: double.infinity,
-              padding: const EdgeInsets.all(15),
-              decoration: BoxDecoration(
-                  border: Border.all(
-                color: kcLightPink,
-                width: 5,
-              )),
-            ),
+    return Stack(
+      children: [
+        Positioned.fill(
+          child: Container(
+            width: double.infinity,
+            height: double.infinity,
+            padding: const EdgeInsets.all(15),
+            decoration: BoxDecoration(
+                border: Border.all(
+              color: kcLightPink,
+              width: 5,
+            )),
```

Comment:
We can remove this border around the screen. This was only for when I was building it.